analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -263,8 +267,8 @@ def book_histos(
     # Only one b-tagged region required
     # The observable is the total transvesre momentum
     # fmt: off
-    df4j1b = df.Filter("Sum(jet_btag[jet_pt_mask]>=0.5)==1")\
-               .Define("HT", "Sum(jet_pt[jet_pt_mask])")
+    df4j1b = df.Filter("Sum(Jet_btagCSVV2[Jet_pt_mask]>=0.5)==1")\
```

Comment:
should this be `>0.5` like at line 132?

Reply:
No, this is done to reflect appropriate cuts in coffea. Changing to the strict condition will introduce discrepancies.

Reply:
I've added small explanation comment in the file

Reply:
thanks!